docs/development/adding-a-package-manager.md

```diff
@@ -111,81 +111,126 @@ Use `updateLockedDependency` to directly update dependencies in lock files.
 
 ## Package files and Lock files
 
-In Renovate terminology, "package files" are the files where human-readable dependency definitions are kept.
-For example, this includes npm's `package.json` file, Maven's `pom.xml` file, and Docker's `Dockerfile`.
+A special Renovate term is "package files".
+Package files have human-readable dependency definitions, for example:
 
-Some package managers may additionally have "lock files", e.g. npm's `package-lock.json`.
-If a lock file is present in a repository then Renovate needs to update both in the same commit, otherwise the update may be "broken".
-Therefore if a new manager is being developed and it is usual to have a lock file, supporting lock file updating should be done from the start.
+- npm's `package.json` file
+- Maven's `pom.xml` file
+- Docker's `Dockerfile`
 
-Supporting lock file updating usually requires Renovate to support a third party tool, e.g. `npm`, `poetry`, etc.
-It's rare and not recommended for Renovate to "reverse engineer" lock file formats and make updates manually instead of calling such tools.
-Adding support for such tools requires adding awareness of each tool to [Containerbase](https://github.com/containerbase/base) first.
+Some package managers may also have "lock files", like npm's `package-lock.json`.
+
+If a repository has a lock file, then Renovate MUST update the package file _and_ the matching lock file in the same commit.
```

Comment:
```suggestion
If a repository has a lock file, then Renovate _must_ update the package file _and_ the matching lock file in the same commit.
```

If we want to emphasize this point, we should use _italics_ to do so. 😉 Uppercase text is harder to read than lowercase text.